### DISCUSSION TITLE
I the Orange Pi Plus 2e Supported ?

### DISCUSSION THREAD

**Discussion by miggi** [Pi4J/pi4j#309] [General]
Hi all, 

I've already seen  questions about Orange Pi 5 support.  But I will try with mine :) 

I'm trying to setup pi4j to  Orange Pi 2E Plus. I'm constantly getting: 
"java.lang.UnsatisfiedLinkError: Pi4J was unable to extract and load the native library"

Is it because that pi4j mostly based on Raspberry Pi right ? or maybe some chance to get it to work. 

Thanks !

**Comment by taartspi** [comment]
Sorry but the conversation in #296 also applies to the 2E.   The Pi4J team has no such plans but that #296 post explains the V2 code design allows a person to add the support.

**Reply by miggi** [reply]
Okay, thank you for you feedback. Will look into #296 .
